tests/test_google_ads_interrupted_sync_full_table.py

```diff
@@ -0,0 +1,122 @@
+from asyncio import streams
+import os
+
+from tap_tester import menagerie, connections, runner
+
+from base import GoogleAdsBase
+
+
+class InterruptedSyncFullTableTest(GoogleAdsBase):
+    """Test tap's ability to recover from an interrupted sync for FULL Table stream"""
+
+    @staticmethod
+    def name():
+        return "tt_google_ads_interruption_full_table"
+
+    def test_run(self):
+
+        # call_details, campaign_labels do not have an id field
+        # accessible_bidding_strategies, accounts, user_list, and bidding_strategies streams have only 1 record.
+        # campaign_criterion, ad_group_criterion have composite primary keys as ad_group_id, criterion_id and campaign_id, criterion_id respectively.
```

Comment:
Remove this line as a test is not skipping the `campaign_criterion` and `ad_group_criterion` streams.

Reply:
Removed unnecessary comments.